src/libraries/Common/tests/System/Collections/IEnumerable.Generic.Tests.cs

```diff
@@ -62,6 +62,18 @@ public abstract partial class IEnumerable_Generic_Tests<T> : TestBase<T>
         /// </summary>
         protected virtual bool Enumerator_Current_UndefinedOperation_Throws => false;
 
+        /// <summary>
+        /// When calling Current of the empty enumerator before the first MoveNext, after the end of the collection,
+        /// or after modification of the enumeration, the resulting behavior is undefined. Tests are included
+        /// to cover two behavioral scenarios:
+        ///   - Throwing an InvalidOperationException
+        ///   - Returning an undefined value.
+        ///
+        /// If this property is set to true, the tests ensure that the exception is thrown. The default value is
+        /// false.
+        /// </summary>
+        protected virtual bool Enumerator_Empty_Current_UndefinedOperation_Throws => Enumerator_Current_UndefinedOperation_Throws;
```

Comment:
Does this need to return the value of `Enumerator_Current_UndefinedOperation_Throws` by default? If so, then the statement in the xml docs above that "the default value is false" is not entirely accurate.
```suggestion
        /// If this property is set to true, the tests ensure that the exception is thrown. The default value is
        /// false.
        /// </summary>
        protected virtual bool Enumerator_Empty_Current_UndefinedOperation_Throws => false;
```